cmd/githubgen/codeowners.go

```diff
@@ -30,7 +30,7 @@ const allowlistHeader = `# Code generated by githubgen. DO NOT EDIT.
 #
 
 ## 
-# NOTE: New components MUST have a codeowner. Add new components to the CODEOWNERS file instead of here.
+# NOTE: New components MUST have a codeowners. Add codeowners to the component metadata.yaml and run make gengithub
```

Comment:
Not necessary in this PR if this is autogenerated, but this is grammatically incorrect.
```suggestion
# NOTE: New components MUST have one or more codeowners. Add codeowners to the component metadata.yaml and run make gengithub
```